### PR TITLE
chore(rpa): improve docs about RPA error handling

### PR DESCRIPTION
**Pull request by mboskamp**
Related to CAM-13012

### CODE REVIEW

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 1]
```diff
@@ -322,22 +322,40 @@ The bot might return more variables which will be ignored in this case.
...
     <camunda:properties>
       <camunda:property name="bot" value="PrintReceipt" />
     </camunda:properties>
   </bpmn:extensionElements>
 </bpmn:serviceTask>
 ```
 
+**How can I match a substring?**
+
+Note that multiple errorEventDefinitions are provided. The first that evaluates to true will trigger the referenced error. In this example `myErrorEventDefinition1` and `myErrorEventDefinition2` will trigger the same error while `myErrorEventDefinition1` will trigger a different error:
```
Comment (tmetzke) [inline content/user-guide/camunda-bpm-rpa-bridge.md:347]:
```suggestion
Note that if multiple errorEventDefinitions are provided, the first one that evaluates to true will trigger the referenced error. In this example `myErrorEventDefinition1` and `myErrorEventDefinition2` will trigger the same error while `myErrorEventDefinition3` will trigger a different one:
```

File: content/user-guide/process-engine/expression-language.md [hunk 2]
```diff
@@ -179,6 +179,39 @@ a bean.
...
 
+## External Task Error Handling
+
+For External Tasks (including tasks handled by [Camunda Platform RPA Bridge]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md" >}})) it is possible to define
+[camunda:errorEventDefinition]({{< ref "/reference/bpmn20/custom-extensions/extension-elements.md#erroreventdefinition" >}})
+elements which can be provided with a JUEL expression. The expression is evaluated on `externalTask.complete()` and
+`externalTask.failed()`. If the expression evaluates to true a BPMN error is thrown which can be caught by an
+[Error Boundary Event]({{< ref "/reference/bpmn20/events/error-events.md#error-boundary-event" >}}).
+
+Within the scope of an External Task expressions have access to the ExternalTaskEntity object via the key `externalTask` which provides getter methods
```
Comment (tmetzke) [inline content/user-guide/process-engine/expression-language.md:190]:
```suggestion
In the scope of an External Task, expressions have access to the `ExternalTaskEntity` object via the key `externalTask` which provides getter methods
```

**Review by tmetzke — COMMENTED** [review]
Looks good, I like that we have more in-depth and quite practical examples. 👍 
Just some minor suggestions wrt. grammar and such.

File: content/user-guide/process-engine/expression-language.md [hunk 3]
```diff
@@ -179,6 +179,39 @@ a bean.
...
+for `errorMessage`, `errorDetails`, `workerId`, `retries` and more.
+
+**Examples:**
+
+How to access the External Task object:
+
+```xml
+<bpmn:serviceTask id="myExternalTaskId" name="myExternalTask" camunda:type="external" camunda:topic="myTopic">
+  <bpmn:extensionElements>
+    <camunda:errorEventDefinition id="myErrorEventDefinition" errorRef="myError" expression="${externalTask.getWorkerId() == 'myWorkerId'}" />
```
Comment (volkergersabeck) [inline content/user-guide/process-engine/expression-language.md:200]: where could I find documentation of the interface of the 'externalTask' object? Could we add a link to that?

Comment (mboskamp) [reply content/user-guide/process-engine/expression-language.md:200]: Fair point. I added a link to the JavaDoc of the interface with #840

**Pull request merged**